homeassistant/components/recorder/auto_repairs/schema.py

```diff
@@ -101,9 +101,7 @@ def _validate_table_schema_has_correct_collation(
 
         collate = (
             dialect_kwargs.get("mysql_collate")
-            or dialect_kwargs.get(
-                "mariadb_collate"
-            )  # pylint: disable-next=protected-access
+            or dialect_kwargs.get("mariadb_collate")  # pylint: disable-next=protected-access
```

Comment:
`disable-next` seems a bit oddly placed now, this comment should move a bit.

Reply:
I think this is correct – the next line has the `_fetch_setting` private use (just like it did before).

Reply:
Right, but we can place it on its own line. Right now it implies a bit the comment is about the line it is on :)